day_7/checker_board.js

```diff
@@ -0,0 +1,10 @@
+function checkerBoard (checkers){
+  for (var i = 1; i <= checkers; i++) {
+    if (i % 2 === 0){
```

Comment:
You may need a loop within your loop if you want the ability to change the length of your rows.